### PR TITLE
Add docs on Quarkus functions

### PR DESCRIPTION
**Pull request by rh-max**
@matejvasek, @abrennan89  Could you PTAL?

### CODE REVIEW

File: modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc
Comment (matejvasek) [inline modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc:91]: There is a small difference though. The `withCloudEvent` function cannot be invoked by plain (non-CE) HTTP. Only the last two examples for `withBeans` are valid here. The first two wouldn't work.

**Comment by matejvasek** [comment]
Looks good to me. I just wonder why some `*.adoc` have `*.html` counterpart and some doesn't.

File: modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc
Comment (rh-max) [reply modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc:91]: Hmm, I thought specifying "with `CloudEvent`" was enough. However, you are right, it's better to be explicit, so how about:

* The `withCloudEvent` function of the `Functions` class can be invoked with `CloudEvent` similarly to the `withBeans` function. However, unlike `withBeans`, `withCloudEvent` cannot be invoked with a plain HTTP request.

**Comment by rh-max** [comment]
@matejvasek Thanks for the review! Removed the rogue `.html`s, I added them accidentally.

**Review by abrennan89 — APPROVED** [review]

**Comment by abrennan89** [comment]
@rh-max thanks Max, I'll merge this now but we will need to run antora to make the site update. I can do this later but FYI there are instructions here https://github.com/openshift-knative/docs#building-the-site

**Comment by rh-max** [comment]
@abrennan89 Oops, a bit hasty, I haven't commited the small suggestion above (was waiting for Matěj's approval) :) Should I do another PR or do you have some general PR where we can insert this change?

File: modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc
Comment (matejvasek) [reply modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc:91]: I think that suggested change is good.

Comment (rh-max) [reply modules/ROOT/pages/functions/dev_guide/quarkus/invocation-examples-quarkus.adoc:91]: Thanks Matěj, applied the suggestion in https://github.com/openshift-knative/docs/pull/62

**Pull request merged**
